src/kernel/task.zig

```
@@ -254,3 +308,75 @@ test "allocatePid and freePid" {
 
     expectEqual(all_pids.bitmap, 0);
 }
+
+test "addVFSHandle" {
+    var task = try Task.create(0, true, undefined, std.testing.allocator);
+    defer task.destroy(std.testing.allocator);
+    var node1 = vfs.Node{ .Dir = .{ .fs = undefined, .mount = null } };
+    var node2 = vfs.Node{ .File = .{ .fs = undefined } };
+
+    const handle1 = task.addVFSHandle(&node1) orelse unreachable;
```

Maybe return an error instead of unreachable, as if the test are compiled as release-fast/small, then will have undefined behavior.